docs/erwin-goals.md

```diff
@@ -0,0 +1,20 @@
+II a dBildung-IAM as a production ready, stand-alone system
```

Comment:
This numbering might be confusing, if someone reads this without access to the Confluence article. Maybe, make a headline out of it?